src/main/scala/wdl4s/AstTools.scala

```diff
@@ -195,8 +195,8 @@ object AstTools {
    * key/value pairs (if the input section is specified).  This will then return a Seq of
    * IOMapping(key=<terminal> value=<expression ast>)
    *
-   * @param ast
-   * @param wdlSyntaxErrorFormatter
+   * @param ast The call AST
+   * @param wdlSyntaxErrorFormatter The wdl syntax error formatter
```

Comment:
I'd advocate for just doing away with these as I think the names are clear enough (particularly w/ the proposed definitions). However actually *having* definitions is an improvement over what's there before so feel free to disregard